docs/platforms/javascript/guides/deno/configuration/integrations/breadcrumbs.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Breadcrumbs
+excerpt: ""
+description: "Wraps native APIs to capture breadcrumbs. (default)"
+sidebar_order: 1
+notSupported:
+  - javascript.electron
+---
+
+_Import name: `Sentry.breadcrumbsIntegration`_
+
+This integration is [enabled by default](./../#modifying-default-integrations).
+
+This integration wraps native APIs to capture breadcrumbs. By default, the Sentry SDK wraps all APIs.
+You can opt-out of capturing breadcrumbs for specific parts of your application (e.g. do not capture `console.log` calls as breadcrumbs) via the options below.
+
+In
```

Comment:
```suggestion

When you're using Sentry's SDK for Deno, the breadcrumbs-capture integration wraps native APIs to capture breadcrumbs. Because the Sentry SDK wraps all APIs, this integration is enabled by default. Read about how to modify default integrations [here](./../#modifying-default-integrations).

You can opt out of capturing breadcrumbs for specific parts of your application (for example, you could say don't capture `console.log` calls as breadcrumbs) via the options below.

```
We try to stay away from Latin abbreviations (eg, etc,...) 
We also use contractions as much as possible (do not > don't)
:)

Reply:
I actually find it useful to have the information that this is a default integration at the top :) I also talked to @lforst and he also thinks that having that really short at the top is better.

What do you think of that:

> This integration is enabled by default. You can also [modify default integrations](./../#modifying-default-integrations).
> 
> The `breadcrumbsIntegration` wraps native APIs to capture breadcrumbs. By default, the Sentry SDK wraps all APIs. You can opt out of capturing breadcrumbs for specific parts of your application (for example, you could say don't capture `console.log` calls as breadcrumbs) via the options below.

As all other pages with default integrations have this first line "This integration is enabled by default", I would modify the "base page" for integrations too in another PR -  if my suggestion is alright :)